js/ui/handler/touch_zoom_rotate.js

```diff
@@ -39,6 +39,13 @@ TouchZoomRotate.prototype = {
         this._startVec = p0.sub(p1);
         this._startScale = this._map.transform.scale;
         this._startBearing = this._map.transform.bearing;
+        this._scalingSignificantly = false;
+        this._rotatingSignificantly = false;
+        this._blockRotation = undefined;
```

Comment:
It is a little funny to have both `_blockRotation ` and `_rotationDisabled` properties in the same class. Renaming `_blockRotation` to emphasize its relationship to `_rotatingSignificantly` would make the code more clear.